js/source/legacy/CXGN/BreedersToolbox/UploadPhenotype.js

```diff
@@ -197,6 +197,7 @@ function displayPhenotypeUploadVerifyResponse(response, upload_type) {
             message_text += "<span class='badge'><span class='glyphicon glyphicon-asterisk'></span></span>";
             message_text += "Warnings are shown in yellow. Either fix the file and try again or continue with storing the data.";
             message_text += "<hr>To overwrite previously stored values instead: <input type='checkbox' id='phenotype_upload_overwrite_values' name='phenotype_upload_overwrite_values' />";
+	    message_test += "Note: if overwrite is not clicked, not values will be added to the database, even for non repeated traits. ";
```

Comment:
Should this be `message_text`?  And should `not values` be `new values`?

Reply:
Also, I think when you upload a 2nd file containing a single trait that's already in the database, the 2nd set of new values is NOT added to the database, but is entirely skipped.